capa/main.py

```diff
@@ -857,6 +920,10 @@ def main(argv: Optional[List[str]] = None):
         except ShouldExitError as e:
             return e.status_code
 
+        if any(extractor_filters.values()):
+            # if the user specified any extractor filters, apply them here.
+            extractor = apply_extractor_filters(extractor, extractor_filters)
```

Comment:
i think this should be moved within `get_extractor_from_cli` so that all callers benefit from this filtering capability

Reply:
also, the `if any(...)` should go within `apply_extractor_filters` so that we can always call the routine unconditionally. this requires less cognitive overhead for readers/users.

Reply:
I have done this now. I am now getting the filters and applying them from within the `get_extractor_from_cli`.